### PR TITLE
Rework fastapi-bootcamp app with newer starter

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

**Comment by athornton** [comment]
Ready for review.

File: applications/fastapi-bootcamp/templates/ingress.yaml [hunk 1]
```diff
@@ -22,7 +21,7 @@ template:
       - host: {{ required "global.host must be set" .Values.global.host | quote }}
         http:
           paths:
-            - path: {{ .Values.config.pathPrefix | quote }}
+            - path: "/fastapi-bootcamp"
```
Comment (athornton) [inline applications/fastapi-bootcamp/templates/ingress.yaml:24]: Sorry, Jonathan.  Take it up with Russ.

Comment (rra) [reply applications/fastapi-bootcamp/templates/ingress.yaml:24]: ...that's an odd way of saying "hey, I think there's a bug in the starter." I'll go fix the starter; I added the config setting but then forgot to use it.

**Review by rra — APPROVED** [review]
That all looks good except for the ingress path change, which you can revert (the starter should be fixed shortly).

File: applications/fastapi-bootcamp/templates/ingress.yaml (hunk 1 above)
Comment (athornton) [reply applications/fastapi-bootcamp/templates/ingress.yaml:24]: Oh, the history behind that is that on the first iteration of that, I had "/fastapi-bootcamp", and Jonathan caught it.  I figured this was a disagreement over how opinionated we wanted to be.

**Pull request merged**